apps/crn-server/src/data-providers/research-outputs.data-provider.ts

```diff
@@ -76,24 +77,37 @@ export class ResearchOutputSquidexDataProvider
   }): Promise<ListResearchOutputDataObject> {
     const { search, filter, take = 8, skip = 0, includeDrafts } = options;
 
-    const searchQ = (search || '')
+    const containsFilters = (search || '')
       .split(' ')
       .filter(Boolean)
-      .map(sanitiseForSquidex)
       .reduce(
-        (acc: string[], word: string) =>
-          acc.concat(
-            `contains(data/title/iv, '${word}') or contains(data/tags/iv, '${word}')`,
-          ),
-        [],
-      )
-      .join(' or ');
+        (res, word: string) => [
+          ...res,
+          {
+            [`data/title/iv`]: {
+              contains: word,
+            },
+          },
+          {
+            [`data/tags/iv`]: {
+              contains: word,
+            },
+          },
+        ],
+        [] as Filter[],
+      );
 
-    const filterQ = makeODataFilter(filter);
+    const searchQ = containsFilters.length
+      ? containsFilters.length === 1
+        ? containsFilters[0]
```

Comment:
there is some funky nested ternary fun happening here and as codecov suggests it is not fully tested. Perhaps you would like to add a test for when this should evaluate to { or ... }?